src/navigations/root-stack.js

```diff
@@ -97,6 +100,12 @@ export const RootNavigator = () => {
     };
   }, []);
 
+  React.useEffect(() => {
+    if (localDb) {
+      resendPendingMessages(localDb, sendChatAnonMutation, sendChatSignedMutation);
+    }
+  }, [localDb]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This useEffect hook is used to resend pending messages when `localDb` is available. It's good practice to check if the mutation functions `sendChatAnonMutation` and `sendChatSignedMutation` are also defined before calling `resendPendingMessages`. This will prevent potential runtime errors if these functions are not properly initialized.

```diff
- if (localDb) {
+ if (localDb && sendChatAnonMutation && sendChatSignedMutation) {
    resendPendingMessages(localDb, sendChatAnonMutation, sendChatSignedMutation);
  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->